pegasus_wrapper/scripts/example_container_workflow_builder.py

```diff
@@ -32,6 +36,11 @@ def example_workflow(params: Parameters):  # pragma: no cover
     An example script to generate a workflow for submission to Pegasus.
     """
     tmp_path = params.creatable_directory("example_root_dir")
+    docker_shared_volume = params.string("docker_shared_volume", default="/data/")
+    docker_nas_volume = params.string(
+        "docker_nas_volume", default="/scratch/dockermount/pegasus_wrapper_demo"
+    )
```

Comment:
I'm not clear on the use of 'docker_nas_volume' as a name when the default value /scratch is not nas but local storage.  Maybe the variable could have a different name, 'docker_final_storage' or something clearer.